src/applications/education-letters/components/LoginWidget.jsx

```diff
@@ -45,9 +45,11 @@ const LoginInWidget = ({ toggleLoginModal, user }) => {
         </a>{' '}
         account now.
       </div>
-      <button className="va-button-primary" type="button" onClick={toggleLogin}>
-        Sign in or create an account
-      </button>
+      <va-button
+        onClick={toggleLogin}
+        primary-alternate
+        text="Sign in or create an account"
```

Comment:
Beacuse this is a sign-in button, we can use the `primary-alternate` prop which has a green background:

https://design.va.gov/storybook/?path=/docs/components-va-button--primary-alternate